index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Brian and Michael Personal Portfolio webpage</title>
+    <link rel="stylesheet" href="styles.css"/>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul id="nav-list">
+        <li>
+          <a href="welcome-section">Information<a/>
+        </li>
+        <li>
+          <a href="#projects">Activities<a/>
+        </li>
+        <li>
+          <a href="#contact">Contact us<a/>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey, We are Brian and Michael</h1>
+      <p>We are upcoming Web Developers</p>
+    </section>
+    <section id="projects" class="projects">
+      <h2>We are currently working on this Project</h2>
+        <div class="project-grid">
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
+            <p class="project-tile">The Story of President Barrack Obama</p>
+          </a>
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="" class="project-img"/>
+            <p class="project-tile">President Barack Obama taking his Oath of Office</p>
+          </a>
+          
+        </div>
```

Comment:
Please do not leave a blank space in this block of code. The blank space in this block of code is unnecessary. To make your code cleaner and easier to read, it is better to leave space only between complete blocks of code, and not within each block.